shared/config/superblocks.ts

```diff
@@ -75,10 +76,10 @@ export const superBlockOrder: SuperBlockOrder = {
     SuperBlocks.CodingInterviewPrep,
     SuperBlocks.ProjectEuler
   ],
+  [SuperBlockStages.VisibleLegacy]: [SuperBlocks.JsAlgoDataStruct],
```

Comment:
I would prefer to just put this in the existing legacy and show the RWD with this - the Python for Everybody will be in there too.